newrelic/config.py

```diff
@@ -443,6 +443,7 @@ def _process_configuration(section):
     )
     _process_setting(section, "custom_insights_events.enabled", "getboolean", None)
     _process_setting(section, "custom_insights_events.max_samples_stored", "getint", None)
+    _process_setting(section, "custom_insights_events.max_attribute_value", "getint", 255)
```

Comment:
Should we modify the setting name to clarify that this is impacting the attribute length specifically? Maybe `custom_insights_events.max_attribute_length`?

Reply:
Prob a discussion to be had on the spec: https://source.datanerd.us/agents/agent-specs/pull/648. We do need to be specific that this is the value not the key though.

Reply:
Yeah maybe `custom_insights_events.max_attribute_value_length`? We can move the discussion to the spec.